### DISCUSSION TITLE
How To Add Downloaded Scenery?

### DISCUSSION THREAD

**Discussion by jcurbaniak** [OpenRCT2/OpenRCT2#22086] [Q&A]
I have downloaded a couple of scenery items: a lamp and a spinning sign.  

They do not show up when the scenery tab is selected during the game.

I have the items in BOTH:

D:\GOG Galaxy\Games\RollerCoaster Tycoon 2 Triple Thrill Pack\ObjData 

and

C:\Users\Jerry New\Documents\OpenRCT2\object

What am I doing wrong? Thanks in advance.

**Comment by spacek531** [comment]
Hello,

Custom objects should go in OpenRCT2\Object not ObjData. You should remove the copies from ObjData.

All scenarios have specific scenery that is available that is set when the scenario is authored. You can change this during gameplay using the Object Selection cheat.

**Comment by jcurbaniak** [comment]
Thanks.  Finally found the sign in Object Selection.  Turns up under a ? tab in the actual Scenery selection.

 

Jerry U.
